README.md

```diff
@@ -201,6 +216,29 @@ Start and serve the dApp:
 npm run start
 ```
 
+### Hot Reload Contract Address 
+
+By default, every page refresh will deploy new contract and reset contract data
```

Comment:
This sentence is not really accurate. A page refresh will not perform the deployment per se. It updates the page UI to point to the address of the new deployment.

Reply:
Thank you, Jimmy. Good point.

Reply:
I'll fix this in a subsequent PR.